quickstarts/mlops/superwise/dashboards/superwise.json

```diff
@@ -0,0 +1,273 @@
+{
+  "name": "Superwise - ML Models Overview",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```

Comment:
Permissions are not required so we can remove this and it should pass validation. The permission field is present in the JSON when you copy a dashboard, but the creation schema needs you to remove it, even in the platform
```suggestion
```